src/praisonai-sandbox/praisonai_sandbox/compute/flyio.py

```
@@ -0,0 +1,279 @@
+    async def upload_file(
+        self, instance_id: str, local_path: str, remote_path: str,
+    ) -> bool:
+        # Fly.io file upload via exec + base64
+        import base64
+        try:
+            with open(local_path, "rb") as f:
+                data = base64.b64encode(f.read()).decode()
+            cmd = f"echo '{data}' | base64 -d > {remote_path}"
+            result = await self.execute(instance_id, cmd)
+            return result["exit_code"] == 0
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)